routes/api/users.js

```diff
@@ -129,4 +133,37 @@ router.patch(
   }
 );
 
+router.get("/verify/:verificationToken", async (req, res, next) => {
+  const { verificationToken } = req.params;
+  try {
+    console.log("verificationToken:", verificationToken);
+    const verifiedUser = await verifyUser(verificationToken);
+    console.log("verifiedUser: ", verifiedUser);
```

Comment:
Te console.log'i raczej do usunięcia, o ile nie ma zapotrzebowanie na logowanie każdej akcji to zostawiamy tylko istotne rzecze np jak przerwanie połączenia z bazą danych, synchronizacja kursów walut itp..